frontend_server/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "docker_web_app",
+  "version": "1.0.0",
+  "description": "Node.js on Docker",
+  "author": "First Last <first.last@example.com>",
```

Comment:
delete/replace this one?

Reply:
Good call! I also updated to make it private, matching how our frontend package.json looks